admin/tabs/activity/tracking/ActivityModel.js

```diff
@@ -0,0 +1,227 @@
+/*
+ * This file belongs to Hoist, an application development toolkit
+ * developed by Extremely Heavy Industries (www.xh.io | info@xh.io)
+ *
+ * Copyright © 2020 Extremely Heavy Industries Inc.
+ */
+import {isEmpty, isFinite} from 'lodash';
+import {usernameCol} from '@xh/hoist/admin/columns';
+import {dateTimeCol, GridModel} from '@xh/hoist/cmp/grid';
+import {HoistModel, LoadSupport, managed, XH} from '@xh/hoist/core';
+import {fmtDate, numberRenderer} from '@xh/hoist/format';
+import {action, bindable, comparer, observable} from '@xh/hoist/mobx';
+import {LocalDate} from '@xh/hoist/utils/datetime';
+import {DimensionChooserModel} from '@xh/hoist/cmp/dimensionchooser';
+import {Cube} from '@xh/hoist/data';
+import {ChildCountAggregator, LeafCountAggregator, RangeAggregator} from '../aggregators';
+import {ChartsModel} from './charts/ChartsModel';
+
+@HoistModel
+@LoadSupport
+export class ActivityModel {
+
+    @bindable.ref startDate = LocalDate.today().subtract(7);
+    @bindable.ref endDate = LocalDate.today().add(1);  // https://github.com/xh/hoist-react/issues/400
```

Comment:
Changing the server-side to filter based on the *day* is going to solve this cleanly.